gruntfile.js

```diff
@@ -146,5 +153,5 @@ module.exports = function(grunt) {
     });
 
     grunt.registerTask('serve', ['showcase', 'less:development', 'connect:livereload', 'openfin:serve', 'watch']);
-    grunt.registerTask('build', ['openfin:build', 'showcase']);
+    grunt.registerTask('build', ['showcase', 'less:development', 'connect:livereload', 'download', 'openfin:serve', 'watch']);
 };
```

Comment:
One more small thing:

Perhaps the `build` task could call the `serve` task? There seems to be a lot of duplication between the two.